### PR TITLE
Removes task dependencies in deploy when publishing snapshots

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Good idea from @shakuzen, we shouldn't need to run depedencies of deploy
as they are done prior.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -62,7 +62,7 @@ pipeline {
                 branch 'master'
             }
             steps {
-                sh './mvnw deploy -Papache-release -Dgpg.skip=true -DskipTests --batch-mode -pl -:brave-itests -nsu'
+                sh './mvnw deploy:deploy -Papache-release -Dgpg.skip=true -DskipTests --batch-mode -pl -:brave-itests -nsu'
```
Comment (shakuzen) [inline Jenkinsfile]: `-DskipTests` becomes a bit superfluous since the test phase shouldn't be run anyways.

**Review by abesto — COMMENTED** [review]
We should also be able to drop `-DskipTests` with this, non?

**Review by shakuzen — APPROVED** [review]
LGTM

File: Jenkinsfile [hunk 2]
```diff
@@ -62,7 +62,7 @@ pipeline {
                 branch 'master'
             }
             steps {
-                sh './mvnw deploy -Papache-release -Dgpg.skip=true -DskipTests --batch-mode -pl -:brave-itests -nsu'
+                sh './mvnw deploy:deploy -Papache-release -Dgpg.skip=true --batch-mode -pl -:brave-itests -nsu'
```
Comment (shakuzen) [inline Jenkinsfile:65]: I wonder if `-nsu` has any meaning anymore also. Not sure.

**Pull request merged**